### PR TITLE
[Semi-Modular] Adds the Ranged Analyzer from Oldbase

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Adds the Ranged Analyzer, a semi-late analyzer that can scan any tile/item in sight rather than only surrounding the user.
The CE starts with one in his toolbelt to fit their theme of starting with advanced tools.

Also, to fit our new tool sprites better, I've taken Bee's icon for the tool (much like we did for our other tools) - it's a white version of the original analyzer.
image

<details><summary>**Technical Info:**
</summary>

So, due to TG's habit of hardcoding, the analyzer's code is forced into looking at the user's location. This required the TG proc `/obj/item/analyzer/attack_self(mob/user)` to be overridden; it's been _mostly_ copy-pasted into the new ranged_analyzer.dm, being replaced by `/obj/item/analyzer/proc/scan_turf(mob/user, turf/location)`, which itself is called in replacement of the end of `attack_self()`

TL;DR, the bottom-end of `attack_self()` now calls our new replacement proc, `scan_turf()`
Other non-modular changes include the Ranged Analyzer's addition to the CE belt, and to the Experimental Tools techweb

Also, fun fact? The original code had no sight restrictions; you could scan through walls. That's been fixed.
</details>
</details>

## How This Contributes To The Skyrat Roleplay Experience
A new unique tool, that we had on oldbase.
Sometimes you want to know how hot that plasmafire is, but dont exactly want to test it in-person. Doing it from range will be much nicer.

## Changelog

:cl: SolGov Engineering Safety Association (SESA)
add: Congratulations, Nanotrasen Space Station! Your engineering crew has passed safety inspections with a grade of "C-", clearing you for use of spectroscopy/infrared based Atmospheric Analyzers! These Ranged Analyzers should be a large improvement to the standard.
/:cl:

### CODE REVIEW

**Review by GoldenAlpharex — APPROVED** [review]
Looks alright code-wise, unsure of the balance implications, but the again, it's literally just scanning gas.

**Comment by ghost** [comment]
> 
> 
> Looks alright code-wise, unsure of the balance implications, but the again, it's literally just scanning gas.

Balance should be fine considering that not only is it just scanning gas, but its been nerfed from oldbase considering it cant scan unless you can see the tile now 👍

**Review by Gandalf2k15 — APPROVED** [review]
